cocos/gfx/base/define.ts

```diff
@@ -1610,16 +1614,16 @@ export class SubpassDependency {
         public srcSubpass: number = 0,
         public dstSubpass: number = 0,
         public generalBarrier: GeneralBarrier = null!,
-        public prevAccesses: AccessFlags[] = [AccessFlagBit.NONE],
-        public nextAccesses: AccessFlags[] = [AccessFlagBit.NONE],
+        public prevAccesses: AccessFlags = AccessFlagBit.NONE,
+        public nextAccesses: AccessFlags = AccessFlagBit.NONE,
```

Comment:
The type of `nextAccesses` and `prevAccesses` was changed from `Array< AccessFlags >` to `AccessFlags`, any reasons? Would it break the compatibility?

Reply:
Synchronization related features are not available in web. It is only used by vulkan.
I think it will not break the compatibility.